packages/vite/docs/test-examples.md

```diff
@@ -3,6 +3,12 @@ title: Examples for the Vitest executor
 description: This page contains examples for the Vitest @nx/vite:test executor.
 ---
 
+{% callout type="warning" title="Deprecated" %}
+The `@nx/vite:test` executor is deprecated. Please use `@nx/vitest:test` instead. This executor will be removed in Nx 23.
+
+See the [migration guide](/packages/vitest/documents/migrating-from-nx-vite) for more information.
```

Comment:
It's better to use an actual guide rather than API docs. e.g. `/docs/technologies/testingtools/vitest/guides/migrating-from-nx-vite`

The API docs setup is fine for generators/executors/migrations examples, but let's avoid it for other use cases.